thoth-core/src/test/java/fr/maif/eventsourcing/impl/KafkaEventPublisherTest.java

```diff
@@ -196,7 +196,7 @@ public void eventConsumptionWithEventFromDb() throws IOException {
         assertThat(events).hasSize(6);
 
         verify(eventStore, times(1)).markAsPublished(any(), Mockito.<List<EventEnvelope<TestEvent, Void, Void>>>any());
-        verify(eventStore, times(1)).markAsPublished(Mockito.<List<EventEnvelope<TestEvent, Void, Void>>>any());
+        verify(eventStore, atLeastOnce()).markAsPublished(Mockito.<List<EventEnvelope<TestEvent, Void, Void>>>any());
```

Comment:
There seems to be a race condition here as well, perhaps in some cases akka loads events from DB in two batchs ?